brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-1092 Info Box: Icon/Image padding not working properly

### Description
When we drag-drop infobox and try to apply padding to the icon element the icon size changes. Padding applied to the wrong selector we need to fix this.

Also, the left & right padding is not similar when applied to the icon - https://share.getcloudapp.com/BluG40AK

### Screenshots
 https://share.getcloudapp.com/BluG40AK

### Types of changes

Bug fix



### How has this been tested?

- Drag-drop infobox
- Apply padding to icon element in the style panel
- Check if we add padding the icon size remains the same or not
- Update the page and check on the front-end

### Checklist:
